.github/workflows/tests.yaml

```diff
@@ -152,6 +152,8 @@ jobs:
     if: false
     strategy:
       fail-fast: false
+      matrix:
+        shard_number: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
(blocking) Go ahead and remove this for now. The Azure tests are hard coded not to run and if we enable them it will block all PRs if the Azure tests are not all 100% complete. Also there are other parts of the azure section that need to be updated in order to handle the Playwright output. I think I still have a branch that I started to update this. I'll find it and fix it up so that you'll be able to run it.

Reply:
removed